packages/cli/src/util/projects/link.ts

```diff
@@ -150,15 +150,20 @@ export async function getLinkedProject(
       getProjectByIdOrName(client, link.projectId, link.orgId),
     ]);
   } catch (err) {
-    if (err?.status === 403) {
+    if (err instanceof APIError && err.status === 403) {
```

Comment:
I'm not so sure that the error is an instance of `APIError`.

Where is it thrown?

Reply:
Good call. `APIError` is thrown in some cases, but not in the `getOrgById` -> `getUser` path. Updated!

Reply:
Although, those don't have a status of 403 anyway. The end result appears to be the same regardless. Those non-`APIError` errors will get re-thrown and their messages get logged out.